test/functional/helpers/adobeIOClientCredentials.js

```diff
@@ -42,9 +41,7 @@ if (clientSecret && (privateKeyPath || privateKeyContents)) {
   } catch (e) {
     // eslint-disable-next-line no-console
     console.error(
-      chalk.redBright(
-        `Failed to read private key at ${privateKeyPath}. Please ensure the value provided in the ${PRIVATE_KEY_FILE_ENV_VAR_NAME} environment variable is correct.`
-      )
+      `Failed to read private key at ${privateKeyPath}. Please ensure the value provided in the ${PRIVATE_KEY_FILE_ENV_VAR_NAME} environment variable is correct.`
```

Comment:
Chalk started to offer just ES module format in their latest version. I tried to change this file to become an ES module but it is used by TestCafe and the React app. To make it work with TestCafe, I would need to change all the TestCafe scripts to ES modules. I was entering a rabbit hole so I decided to just remove Chalk usage from this file.